network/src/peer/mod.rs

```diff
@@ -139,7 +140,10 @@ where
     pub async fn start(mut self) {
         let self_peer_id = self.peer_id();
```

Comment:
Let's rename this to `remote_peer_id`. I think `self_peer_id` is super confusing here

Reply:
See: https://github.com/libra/libra/pull/5982